src/main/java/core/basesyntax/Forms.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public enum Forms {

```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Form {

```